### PR TITLE
Hotfix/0.2.2

### PR DESCRIPTION
**Pull request by byb121**
More like a big feature release:

* updated cwl file version for for `chksum` step in workflows to `0.4.0`, some input and output parameter names were changed to reflect the HTTP method was changed to `PUT` from `POST` in the new version;
* suppressed all curl exit codes when putting results.
* output `rg_file_manifest` name is changed to `results_manifest`, schema is changed to:

    ```json
    {
        "input": [
            {"name": "tiny_1.fq.gz", "size": 382, "md5": "...", "sha2": "..."},
            {"name": "tiny_2.fq.gz", "size": 391, "md5": "...", "sha2": "..."}
        ],
        "output": [
            {"name": "interleaved.fq.gz", "size": 614, "md5": "...", "sha2": "..."}
        ]
    }
    ```

* renames some workflow input and output parameter names;
* removed interleaved_report_json output from interleave and paired fastq workflows;
* added a workflow to make a copy of an interleaved fastq and validate its format and base quality score range.

### CODE REVIEW

File: run_checks.sh [hunk 1]
```diff
@@ -4,6 +4,6 @@ echo -e "\n##############################"
 echo      "# Running mdl (markdownlint) #"
 echo      "##############################"
 # disable the line-length check
-mdl -r ~MD013,~MD024 -g .
+mdl -r ~MD013,~MD024,~MD007 -g .
```
Comment (drjsanger) [inline run_checks.sh]: `~MD046` is set in `travis.yml` . Do we want these to match?

Comment (byb121) [reply run_checks.sh]: strangely, without it, on my laptop MD046 was not reported, but I'll keep it consistent.

File: cwls/cgp-seqval-qc_pairs.cwl [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+$namespaces:
+  s: http://schema.org/
+  edam: http://edamontology.org/
+
+s:codeRepository: https://github.com/cancerit/dockstore-cgp-seqval
+s:license: https://spdx.org/licenses/AGPL-3.0
+
+s:author:
+  - class: s:Person
+    s:identifier: https://orcid.org/0000-0002-5634-1539
```
[the remark this answers is not in the document]
Comment (byb121) [reply cwls/cgp-seqval-qc_pairs.cwl]: I've changed email in the `dct:creator` section, which it's on the Dockstore page, into `genservhelp@sanger.ac.uk`. `s:author` section probably is meant to be personal.

[the remark this answers is not in the document]
Comment (drjsanger) [reply cwls/cgp-seqval-qc_pairs.cwl]: I can see arguments both ways. It was just a suggestion

**Pull request merged**